dynatrace/environment_v2/settings.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
+from dynatrace.dynatrace_object import DynatraceObject
+from typing import List, Optional, Union, Dict, Any
+
+from dynatrace.http_client import HttpClient
+from dynatrace.pagination import PaginatedList
+import json
+import logging
+from http.client import HTTPConnection  # py3
+
+
+class SettingService:
+    ENDPOINT = "/api/v2/settings/objects"
+
+    def __init__(self, http_client: HttpClient):
+        self.__http_client = http_client
+    
+    def list(self,schema_id: Optional[str] = None,
```

Comment:
We'll want to update these method names to 'list_objects' etc... The issue would be that this API also lets you list and view 'schemas' so we'd need to name them a bit more clearly to differentiate.

Reply:
renamed methods